### PR TITLE
Feat/restrict hbd to exchange account

### LINKED ISSUE #1441: Restrict sending hbd to exchange account
Sending hbd to exchange account should be restricted

### PR DESCRIPTION
**Pull request by Adesojisouljay**
### What dopes this PR?
This PR restricts the sending of hbd and points to exchange accounts to avoid loss of fund.

[Github issue](https://github.com/ecency/ecency-vision/issues/1441)

Video/screenshot

https://github.com/ecency/ecency-vision/assets/88087890/37cb06a5-1e53-433a-ae67-71d97a9ebe1e

### CODE REVIEW

File: src/common/components/transfer/index.tsx [hunk 1]
```diff
@@ -155,7 +155,8 @@ interface State {
   to: string;
   toData: Account | null;
   toError: string;
-  toExchangeError: string;
+  toExchangeWarning: string;
```
Comment (MTSaeed) [inline src/common/components/transfer/index.tsx:158]: IMO, no need to make an extra state to handle exchange warning.

File: src/common/components/transfer/index.tsx [hunk 2]
```diff
@@ -221,30 +223,37 @@ export class Transfer extends BaseComponent<Props, State> {
...
     this.stateSet({ asset }, () => {
       this.checkAmount();
+      this.exchangeHandler(to, memo);
     });
   };
 
+  exchangeHandler = (to: string, memo: string) => {
+    const { asset } = this.state;
+    
+    if (exchangeAccounts.includes(to) && asset !== "HIVE") {
```
Comment (MTSaeed) [inline src/common/components/transfer/index.tsx:236]: Make a single if to handle **exchangeAccounts.includes(to)** condition.

**Comment by MTSaeed** [comment]
@feruzm Can you please verify me when hover on account name, the styling get destroyed. It's your feature or it's a issue?
I think it's an issue. Kindly verify it's an issue and  then i will resolve this.
Thanks

https://github.com/ecency/ecency-vision/assets/106739598/29d02f07-8d06-43e1-bbc6-fe6a0b2388d8

**Comment by MTSaeed** [comment]
Updated Styling of Modal suggestion list and Search suggestion list:



https://github.com/ecency/ecency-vision/assets/106739598/e2617e4d-8ec9-420e-ae13-5dcd77129d8a

File: src/common/i18n/locales/en-US.json [hunk 3]
```diff
@@ -1222,6 +1222,7 @@
     "memo-help": "This memo is public",
     "memo-error": "Never put private keys into a memo field, it is publicly visible to everyone.",
     "memo-required": "Transfer to an exchange must have a memo.",
+    "invalid-asset": "You can only send hive to an exchange account.",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:1225]: @MTSaeed  make hive => HIVE

File: src/common/components/suggestion-list/_index.scss [hunk 4]
```diff
@@ -143,7 +142,6 @@
         display: flex;
         align-items: center;
         white-space: nowrap;
-        width: 20px;
```
Comment (feruzm) [inline src/common/components/suggestion-list/_index.scss:146]: @MTSaeed double check all usage of suggestion list component and see that it doesn't break in any usage including mobile view.

File: src/common/i18n/locales/en-US.json (hunk 3 above)
Comment (MTSaeed) [reply src/common/i18n/locales/en-US.json]: Done.

**Review by feruzm — APPROVED** [review]

**Pull request merged**